ladybug_comfort/hi.py

```diff
@@ -94,3 +94,44 @@ def heat_index_warning_category(hi):
         category = 4
 
     return category
+
+
+def humidex(ta, tdp):
+    """Calculate Humidex  from air temperature and the Dew Point.
+
+    The Humidex is a Canadian innovation that was first used in 1965.
+    It describes how hot, humid weather feels to the average person.
+    The Humidex combines the temperature and humidity into one number to reflect the perceived temperature.
+    Because it takes into account the two most important factors that affect summer comfort, it can be a better
```

Comment:
Generally a very informative docstring here but I don't think these lines obey our max character limit of 89 characters. If you format them, they should be good to go.

Reply:
Fixed